dev-docs/publisher-api-reference.md

```diff
@@ -724,6 +725,65 @@ pbjs.addAdUnits({
 })
 ```
 
+<a name="adUnit-multiformat">
```

Comment:
I wonder if we still need format-specific examples if we have a good multi-format example?